src/reducers/columnStateHelper.js

```diff
@@ -44,11 +44,14 @@ function initialize(state, props, oldProps) {
   isColumnResizing = props.isColumnResizing !== undefined ? props.isColumnResizing : isColumnResizing;
   columnResizingData = isColumnResizing ? columnResizingData : {};
 
+  const scrollJumped = state.scrollJumped || scrollX != state.scrollX;
```

Comment:
Also considering `state.scrollJumped`, since it can be `true` if there was a row jump.

Reply:
Hmm ya, should we specify if this is related to a change horizontally or vertically.